lib/src/Uncertainty/StatTests/FittingTest.cxx

```diff
@@ -495,7 +495,7 @@ TestResult FittingTest::Kolmogorov(const Sample & sample,
                                    const Scalar level)
 
 {
-  LOGWARN("FittingTest::Kolmogorov(Sample, DistributionFactory, TestResult, level) is deprecated, use BestModelLilliefors");
+  LOGWARN("FittingTest::Kolmogorov(Sample, DistributionFactory, TestResult, level) is deprecated, use Lilliefors");
```

Comment:
Why do you get such modifications in your PR?

Reply:
I missed some details in your branch, commiting it now